controllers/configurationpolicy_controller.go

```diff
@@ -162,41 +173,51 @@ func (r *ConfigurationPolicyReconciler) PeriodicallyExecConfigPolicies(freq uint
 			}
 		}
 
-		//get resources once per cycle to avoid hanging
+		// get resources once per cycle to avoid hanging
 		dd := clientSet.Discovery()
+		//nolint:staticcheck,nolintlint
```

Comment:
```suggestion
		// nolint: staticcheck,nolintlint
```

Reply:
I'm following the example in the docs: https://golangci-lint.run/usage/false-positives/, although both are valid, I think this is best.

> Use `//nolint` instead of `// nolint` because machine-readable comments should have no space by Go convention.

Reply:
I might not have been consistent with this, so I will update it while pulling apart the big commit (and finding what's causing the test failure)